local-dev/api-data-watcher-pusher/Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM alpine:3.11
+FROM python:3-alpine3.11
 
-RUN apk add --no-cache tini bash wget py3-jwt
+RUN apk add --no-cache tini bash wget
+RUN pip install mysql-connector python-keycloak PyJWT
 
 ENV JWTSECRET=super-secret-string \
     JWTAUDIENCE=api.dev
 
 COPY api-watch-push.sh create_jwt.py /home/
```

Comment:
Sorry, forgot this earlier. I would also add the two new python scripts here.

It might not be strictly necessary, unless there's a use case I don't know of or can't remember? This image only gets run locally or on jenkins, and in both cases I'm pretty sure it uses docker compose, which will mount the local files into the container when it runs (so the scripts will be there).

Reply:
Should be resolved in 70dd1e7ef50c945be06552e2cbdb5acf8a5a7e9f